ORStools/common/client.py

```diff
@@ -73,6 +75,14 @@ def __init__(self, provider=None):
             "Authorization": provider["key"],
         }
 
+        settings = QgsSettings()
+        # Read the current value
+        user_agent = settings.value("qgis/networkAndProxy/userAgent")
+        # Set a new value
+        settings.setValue("qgis/networkAndProxy/userAgent", agent)
+        # Reset to old value
+        settings.setValue("qgis/networkAndProxy/userAgent", user_agent)
+
```

Comment:
This needs to be done after the request has been made.

Reply:
Done